### ISSUE TITLE
align releases, with changelog and github-releases

### ISSUE THREAD

**Issue by madmas** [Yermo/nativescript-mapbox#360]
Hi,
currently, the latest version on NPM ist 5.0.1.
The last release listed here on github is 4.4.1 
and the version in src/package.json is 5.0.0

Please try to align these and keep them in sync to users can follow recent developments more easily.
I can stringly suggest the https://github.com/semantic-release/semantic-release plugin to automate this (e.g. after merge to master, the plugin automatically does a release to npm, creates the github release page, etc., automation done e.g. with github actions or travis.)
Let me know if you want help with this :)

**Comment by Yermo** [comment]
Actually, that would be super helpful. I recently took the plugin over and is the first time I've been responsible for a relatively popular project here, so there's a bit of a learning curve for me and I'm still coming up to speed.

**Comment by madmas** [comment]
HI @Yermo,
it's not much to do to get this up and running. The plugin relies on the commit messages in the repo to determine the next version number.

Setup is pretty straight forward, the [Getting Started]( https://github.com/semantic-release/semantic-release/blob/master/docs/usage/getting-started.md#getting-started ) describes how to use the setup script and what info you need to provide.

Do you have anything special in mind you want to set up?

I'm using it with [nativescript-pdf-view](https://github.com/madmas/nativescript-pdf-view) (but there are much better samples regarding the commit messages and the versions released so far.) 
For me, it would trigger a release every time a commit with a triggering message is merged to master. This would result in the fact that for larger change sets, one would be forced to use feature or release-prepare branches to keep in control when/what will be released.

If you want, we can to a chat (text or voice, as you like) to see how I can help further :)

**Comment by Yermo** [comment]
I've finally had a chance to look at the Getting Started guide. This is a really good suggestion. 

I understand continuous integration conceptually but have never set it up. Which CI service would you suggest? Travis?

**Comment by madmas** [comment]
Yes, travis works quite good and there is plenty help available as is has been around for quite a while. Github actions look like a valid alternative.
